card_db/en_us/translation.txt

```diff
@@ -0,0 +1,71 @@
+# Translation Instructions
```

Comment:
@sumpfork So I propose a compromise.  I moved the translation instructions to /card_db and removed it from all the *active* languages.  However, I still kept it in the non-active 'xx' language which is the starting point for any new translation.   That way, to start any new language, we only need to give someone (or they grab) every file in that directory and they are ready to start. 

So now the file is in only two places.  Still a little painful to maintain, but hopefully must less of one.  And I'm hoping the file does not change that much as we go forward.  I also update the maintenance tool to copy the latest version into 'xx'.

It might even be good to have a zipped version of the xx directory downloadable somewhere on the webpage.  So someone scarred of GitHub could just download and start.  Hopefully encouraging more languages.